Sources/ProjectSpec/Config.swift

```diff
@@ -10,10 +10,34 @@ public struct Config: Hashable {
         self.type = type
     }
 
-    public static var defaultConfigs: [Config] = [Config(name: "Debug", type: .debug), Config(name: "Release", type: .release)]
+    public static var defaultConfigs: [Config] = [Config(name: ConfigType.debug.name, type: .debug), Config(name: ConfigType.release.name, type: .release)]
 }
 
 public enum ConfigType: String {
     case debug
     case release
+    
+    public var name: String {
+        rawValue.prefix(1).uppercased() + rawValue.dropFirst()
+    }
+}
+
+public extension Collection where Element == Config {
+    func first(with configVariant: String, for type: ConfigType) -> Config? {
```

Comment:
Could you just add a comment saying `including` will be searched as a whole word
```suggestion
    func first(including configVariant: String, for type: ConfigType) -> Config? {
```